suite-native/module-accounts/src/components/AccountDetailBalance.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { HStack } from '@suite-native/atoms';
+import { CryptoAmountFormatter } from '@suite-native/formatters';
+import { CryptoIcon } from '@trezor/icons';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+
+type AccountDetailBalanceProps = {
+    value: string;
+    networkSymbol: NetworkSymbol;
+    isBalance?: boolean;
+};
+
+export const AccountDetailBalance = ({
+    value,
+    networkSymbol,
+    isBalance = true,
```

Comment:
Actually not. isBalance is saying if the value is already formatted rather than if it is a balance. So it may be false. I renamed this component to `AccountDetailCryptoValue` to avoid this confusion.